day_6/exercises/burrito.js

```diff
@@ -16,11 +16,26 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping(topping) {
+    this.toppings.push(topping)
+  }
+
+  removeTopping(topping) {
```

Comment:
notice that you only have one parameter here. What would be a better name for it based on how you're using it on the line below?